### PR TITLE
[49763] Correct links behavior in How smittestopp works page

### PR DESCRIPTION
**Pull request by nc-kano**

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19.Droid/Views/Settings/SettingsHowItWorksActivity.cs [hunk 1]
```diff
@@ -38,49 +35,44 @@ void Init()
...
             backButton.Click += new SingleClick((sender, args) => Finish()).Run;
         }
 
-        private void FormatLink (TextView textView) {
-            SpannableStringBuilder spannable = new SpannableStringBuilder (textView.TextFormatted);
-            Object[] spans = spannable.GetSpans (0, spannable.Length (), Class.FromType (typeof (URLSpan)));
-            foreach (URLSpan span in spans) {
-                int start = spannable.GetSpanStart (span);
-                int end = spannable.GetSpanEnd (span);
+        private void FormatLink(TextView textView)
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.Droid/Views/Settings/SettingsHowItWorksActivity.cs:46]: Add unit tests for this method

File: NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPage2/SettingsPage2ViewController.cs [hunk 2]
```diff
@@ -60,17 +58,23 @@ void SetTexts()
...
+            
             NSMutableAttributedString text = new NSMutableAttributedString();
-            text.Append(ApplyStylingToText(intro, FontType.FontRegular, 1.28, 16, 22, false));
-            text.Append(ApplyStylingToText(par1Title, FontType.FontBold, 1.28, 16, 22, false));
-            text.Append(ApplyStylingToText(par1Content, FontType.FontRegular, 1.28, 16, 22, false));
-            text.Append(ApplyStylingToText(par2Title, FontType.FontBold, 1.28, 16, 22, false));
-            text.Append(ApplyStylingToText(par2Content, FontType.FontRegular, 1.28, 16, 22, false));
-            text.Append(ApplyStylingToText(par3Title, FontType.FontBold, 1.28, 16, 22, false));
-            text.Append(ApplyStylingToText(par3Content, FontType.FontRegular, 1.28, 16, 22, false));
+            text.Append(ApplyStylingToText(intro, FontType.FontRegular));
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPage2/SettingsPage2ViewController.cs:63]: not getting it, why did you delete the font sizes?

File: NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPage2/SettingsPage2ViewController.cs [hunk 3]
```diff
@@ -89,7 +93,7 @@ public void SetupStyling()
             LeaveController();
         }
 
-        private NSMutableAttributedString ApplyStylingToText(string text, FontType fontType, double lineHeight, int fontSize, int maxFontSize, bool isLink)
+        private NSMutableAttributedString ApplyStylingToText(string text, FontType fontType, double lineHeight = 1.28, int fontSize = 16, int maxFontSize = 22, bool isLink = false)
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPage2/SettingsPage2ViewController.cs:96]: nevermind, you added default :D

**Review by pavshr — CHANGES_REQUESTED** [review]

File: NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPage2/SettingsPage2ViewController.cs (hunk 2 above)
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPage2/SettingsPage2ViewController.cs:63]: they are the same for each paragraph so i've just made them default

File: NDB.Covid19/NDB.Covid19.Droid/Views/Settings/SettingsHowItWorksActivity.cs (hunk 1 above)
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19.Droid/Views/Settings/SettingsHowItWorksActivity.cs:46]: This is purely Android method. we have no environment for such tests

**Review by pavshr — COMMENTED** [review]
OK, then this can be merged

**Review by pavshr — APPROVED** [review]
